src/ring_theory/polynomial/homogeneous.lean

```diff
@@ -264,6 +274,19 @@ begin
     simp [hd] }
 end
 
+@[simp]
+lemma homogeneous_component_add :
+  homogeneous_component n (φ + ψ) = homogeneous_component n φ + homogeneous_component n ψ :=
+by rw [homogeneous_component, linear_map.comp_apply, linear_map.comp_apply, linear_map.comp_apply,
+    linear_map.map_add, linear_map.map_add]
```

Comment:
Am I missing something? Isn't this just proved by `linear_map.map_add _ _ _`, which `simp` already knows how to find?

Reply:
Yes that's very strange, I'm in the bad habit of trusting the "simp can already prove this" linter too much to let me know when i've duplicated a simp lemma, I wonder if its possible to make it fire here